### ISSUE TITLE
Locale with script in it

### ISSUE THREAD

**Issue by dbendy** [i18next/i18next#626]
Question: ideally, I would like to have a language/locale in my translations with a script in it:

```json
{
  "zh-Hant-MO": {
    "translation: {
       ...
     }
   }
}
```

`zh-Hant-MO` is a valid locale according to i18n standards and represents Chinese in traditional script.  Will `i18next` work with this locale properly?  If an app makes a call to the `t` function with `zh-Hant-MO` as the language, will it work?  I have looked through the `i18next` docs and seen examples with languages and countries, i.e. `en` and `en-US`, but nothing with scripts.  Please clarify.  Thank you!

**Comment by jamuhl** [comment]
right now it wouldn't work...but will add support asap

**Comment by jamuhl** [comment]
support now language code variants with scripts and other exotic forms: zh-Hans-MO, sgn-BE-fr, de-AT-1996,...

published version 2.4.0

**Comment by dbendy** [comment]
Thank you!
